I'm still getting the same CMake errors:

```
Make Error at build/_deps/jsonifier-src/CMake/JsonifierDetectArchitecture.cmake:34 (math):
  math cannot parse the expression: "no such file or directory": syntax
  error, unexpected end of file (25).
Call Stack (most recent call first):
  build/_deps/jsonifier-src/CMakeLists.txt:57 (include)


CMake Error at build/_deps/jsonifier-src/CMake/JsonifierDetectArchitecture.cmake:46 (math):
  math cannot parse the expression: "( ERROR & 0x1 )": syntax error,
  unexpected exp_AND (9).
Call Stack (most recent call first):
  build/_deps/jsonifier-src/CMakeLists.txt:57 (include)


CMake Error at build/_deps/jsonifier-src/CMake/JsonifierDetectArchitecture.cmake:50 (math):
  math cannot parse the expression: "( ERROR & 0x2 )": syntax error,
  unexpected exp_AND (9).
Call Stack (most recent call first):
  build/_deps/jsonifier-src/CMakeLists.txt:57 (include)
```

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -572,7 +572,7 @@ auto jsonifier_test()
 
     try {
         for (size_t i = 0; i < iterations; ++i) {
-            parser.parseJson(obj, buffer);
+            parser.parseJson < jsonifier::parse_options{ .minified = true } > (obj, buffer);
             parser.serializeJson(obj, buffer);
         }
     }
@@ -608,7 +608,7 @@ auto jsonifier_test()
     t0 = std::chrono::steady_clock::now();
 
     for (size_t i = 0; i < iterations; ++i) {
-        parser.parseJson<true>(obj, buffer);
+        parser.parseJson < jsonifier::parse_options{ .minified = true } > (obj, buffer);
     }
 
     t1 = std::chrono::steady_clock::now();
@@ -639,7 +639,7 @@ auto jsonifier_abc_test()
     auto t0 = std::chrono::steady_clock::now();
 
     for (size_t i = 0; i < iterations_abc; ++i) {
-        parser.parseJson<true>(obj, buffer);
+        parser.parseJson < jsonifier::parse_options{ .minified = true } > (obj, buffer);
     }
 
     auto t1 = std::chrono::steady_clock::now();

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -564,7 +564,7 @@ auto glaze_abc_test()
 auto jsonifier_test()
 {
     jsonifier::jsonifier_core parser{};
-    std::string buffer{ json0 };
+    std::string buffer{ json_minified };
 
     obj_t obj;
 

